packages/foundation-models/README.md

```diff
@@ -17,9 +17,54 @@ $ npm install @sap-ai-sdk/foundation-models
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
 
-## Usage
+## OpenAI client
 
-<!-- Details to be added -->
+The OpenAI client can be used to send chat completion or embedding requests to the OpenAI model deployed in SAP Generative AI Hub.
```

Comment:
[req] As above, this is not the official name, please use something like "generative AI hub in SAP AI Core" instead.